akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/PersistentBehaviors.scala

```diff
@@ -108,4 +118,11 @@ trait PersistentBehavior[Command, Event, State] extends DeferredBehavior[Command
    * The `tagger` function should give event tags, which will be used in persistence query
    */
   def withTagger(tagger: Event ⇒ Set[String]): PersistentBehavior[Command, Event, State]
+
+  /**
+   * Transform the event in another type before giving to the journal. Can be used to wrap events
+   * in types Journals and Adapters understand as well as simple event migrations.
+   */
```

Comment:
How could it be used to do migration? If you already have the events they won't be adapted? Remove or clarify?

Reply:
Updated